src/tutorial/microbe_editor/StaySmallTutorial.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Tutorial
+{
+    public class StaySmallTutorial : EditorEntryCountingTutorial
+    {
+        public StaySmallTutorial()
+        {
+            CanTrigger = false;
+        }
+
+        public override string ClosedByName => "StaySmallTutorial";
+
+        protected override int TriggersOnNthEditorSession => 3;
```

Comment:
By round three, isn't it a little late to tell the player to not add anything?

Reply:
Key prompt showing doesn't work for the mouse wheel, and also keypad keys don't work, would need for MICROBE_STAGE_HELP_MESSAGE_1, fix that once implemented

maybe the try to stay small tutorial should not activate if the player has 2 or less organelles

Add new editor help menu explaining the main controls of the editor

MICROBE_STAGE_HELP_MENU_AND_ZOOM should use the actual binding for [thrive:input]g_zoom_out[/thrive:input]
 though maybe the ordering of the auto-evo prediction and this tutorial can be changed. I'll add this on my TODO list to open an issue.

Reply:
Deleted my comment text...
anyway I said that the order of the auto-evo prediction and this tutorial could be swapped, but I don't want to front load any more tutorials to the first editor session.

Reply:
The player can't really mess up in the first session, I think, unless they place an early game nucleus or something stupid, so thats fine for me